tests/integration_tests/performance/test_snapshot_perf.py

```diff
@@ -15,6 +15,11 @@
 import host_tools.network as net_tools  # pylint: disable=import-error
 import host_tools.logging as log_tools
 
+# Supported snapshot target versions for the current Firecracker version.
+FIRECRACKER_VERSIONS = {
```

Comment:
These are also available by parsing the binary ci artifacts from S3. One advantage of that approach is that it does not require to update any code and just upload the binary in S3. The downside is that it is not explicitly visible in the code and would require adding logs to help debug failures.

WDYT ?

Reply:
I would also create a framework method that returns this array.

Reply:
Done